backend/src/test/java/com/benine/backend/http/PresetCreationHandlerTest.java

```diff
@@ -0,0 +1,45 @@
+package com.benine.backend.http;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.OutputStream;
+
+import com.sun.net.httpserver.HttpExchange;
+
+import org.junit.Test;
+
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.CameraController;
+import com.benine.backend.camera.Position;
+import com.benine.backend.camera.ipcameracontrol.IPCamera;
+import com.benine.backend.database.DatabasePreset;
+
+import junit.framework.Assert;
+
+@SuppressWarnings("deprecation")
```

Comment:
This deprecation warning can be removed by changing the import statement from `unit.framework.Assert` to `org.junit.Assert`.